### ISSUE TITLE
Google Analytics

### ISSUE THREAD

**Issue by Zaturrby** [augmenting-masterpieces/AugmentM#59]
Add google analytics number

**Comment by JBarnbeck** [comment]
What about cookies? Do we need a cookie consent tool as well? 
This would be an open source solution http://demo.cookieconsent.silktide.com/

**Comment by Zaturrby** [comment]
Good catch, while we're all out of cookies, it seem Google analytics has plenty. It's unfair.

**Comment by Zaturrby** [comment]
It seems that Vimeo is also setting cookies... So we'll indeed need something like it.

**Comment by Zaturrby** [comment]
Opting out of the vimeo cookies doesn't seem an easy possibility. They themselves adopt the policy of: well if you don't want them, don't allow them. There doesn't seem to be a quick way to opt-out of their cookies. I suppose the best way to handle it is to make a clause in our impressum stating the cookie use.  At best we can allow them to opt out of our analytics, or don't apply them at all.

**Comment by JBarnbeck** [comment]
www.augmentingmasterpieces.eu 
UA-33585261-4

This is your tracking code. Copy and paste it into the code of every page that you want to track.

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-33585261-4', 'auto');
  ga('send', 'pageview');

</script>

**Comment by JBarnbeck** [comment]
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-33585261-4', 'auto');
  ga('send', 'pageview');

</script>
